src/Sql/Sql/help/Clear-AzSqlInstanceVulnerabilityAssessmentSettings.md

```diff
@@ -0,0 +1,156 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Sql.dll-Help.xml
+Module Name: Az.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/az.sql/clear-azsqlinstancevulnerabilityassessmentsettings
+schema: 2.0.0
+---
+
+# Clear-AzSqlInstanceVulnerabilityAssessmentSettings
+
+## SYNOPSIS
+Clears the vulnerability assessment settings of a managed instance.
+
+## SYNTAX
+
+### ClearByNameParameterSet (Default)
+```
+Clear-AzSqlInstanceVulnerabilityAssessmentSettings [-ResourceGroupName] <String> [-InstanceName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ClearByParentResourceParameterSet
+```
+Clear-AzSqlInstanceVulnerabilityAssessmentSettings [-InputObject] <AzureSqlManagedInstanceModel>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Clear-AzSqlInstanceVulnerabilityAssessmentSettings** cmdlet removes the vulnerability assessment settings of an Azure SQL Managed Instance.
+Note that you need to run *Enable-AzSqlInstanceAdvancedDataSecurity* cmdlet as a prerequisite for using this cmdlet.
+
+## EXAMPLES
+
+### Example 1 - Clear the vulnerability assessment settings of an Azure SQL managed instance
+```powershell
+PS C:\> Clear-AzSqlInstanceVulnerabilityAssessmentSettings `
+            -ResourceGroupName "ResourceGroup01" `
+            -InstanceName "ManagedInstance01" `
+```
+
+### Example 2 - Clear the vulnerability assessment settings from a managed instance object
+```powershell
+PS C:\> Get-AzSqlInstance `
+            -ResourceGroupName "ResourceGroup01" `
+            -Name "ManagedInstance01" `
+            | Clear-AzSqlInstanceVulnerabilityAssessmentSettings
+```
+
```

Comment:
Please add samples output

Reply:
Clear-AzSqlInstanceVulnerabilityAssessmentSettings doesn't return anything on success

Reply:
if it doesn't - you may want to consider using `bool` for the output type of the [`Clear-AzSqlInstanceVulnerabilityAssessmentSettings`](https://github.com/Azure/azure-powershell/pull/8413/files?file-filters%5B%5D=.cs#diff-632902af0776c8292c91cb56a243ae9dR26) and [`Clear-AzSqlServerVulnerabilityAssessmentSettings`](https://github.com/Azure/azure-powershell/pull/8413/files?file-filters%5B%5D=.cs#diff-10494e091851c462e9f1cd916383f4abR26) cmdlets and add `-PassThru` parameter.

Reply:
These cmdlets are consistent with our other cmdlets (for databases and managed databases) so we will keep it as it is for now. We might change that in the future for all our cmdlets.

Reply:
It's not clear what you mean by the **consistency**  
* if the other cmdlets don't use their output types - you need to deprecate them following the document: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletoutputbreakingchangeattribute. 
* else - please use the output types to return anything on success and update corresponding help files samples.